specs/validator/beacon_node_oapi.yaml

```
@@ -0,0 +1,609 @@
+          $ref: '#/components/responses/InternalError'
+        503:
+          $ref: '#/components/responses/CurrentlySyncing'
+
+  /validator/attestation:
+    get:
+      tags:
+        - MinimalSet
+      summary: "Produce an attestation, without signature."
+      description: "Requests that the beacon node produce an IndexedAttestation, with a blank signature field, which the validator will then sign."
+      parameters:
+      - name: slot
+        in: query
+        required: true
+        description: "The slot for which the attestation should be proposed."
+        schema:
+          type: integer
+      - name: shard
+        in: query
+        required: true
+        description: "The shard number for which the attestation is to be proposed."
+        schema:
+          type: integer
+      responses:
+        200:
+          description: Success response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/IndexedAttestation'
+        400:
+          $ref: '#/components/responses/InvalidRequest'
+        500:
+          $ref: '#/components/responses/InternalError'
+        503:
+          $ref: '#/components/responses/CurrentlySyncing'
+    post:
+      tags:
+        - MinimalSet
+      summary: "Published a signed attestation."
+      description: "Instructs the beacon node to publish a newly signed IndexedAttestation object, to be incorporated into the beacon chain."
+      parameters:
+      - name: attestation
+        in: query
+        required: true
+        description: "An `IndexedAttestation` structure, as originally provided by the beacon node, but now with the signature field completed."
+        schema:
+          $ref: '#/components/schemas/IndexedAttestation'
+      responses:
+        200:
+          $ref: '#/components/responses/Success'
+        400:
+          $ref: '#/components/responses/InvalidRequest'
+        500:
+          $ref: '#/components/responses/InternalError'
+        503:
+          $ref: '#/components/responses/CurrentlySyncing'
+
+components:
+  schemas:
+    pubkey:
+      type: string
+      format: byte
+      pattern: "^0x[a-fA-F0-9]{96}$"
+      description: "The validator's BLS public key, uniquely identifying them. _48-bytes, hex encoded with 0x prefix, case insensitive._"
+      example: "0x1b66ac1fb663c9bc59509846d6ec05345bd908eda73e670af888da41af171505cc411d61252fb6cb3fa0017b679f8bb2305b26a285fa2737f175668d0dff91cc"
+    version:
+      type: string
+      description: "A string which uniquely identifies the client implementation and its version; similar to [HTTP User-Agent](https://tools.ietf.org/html/rfc7231#section-5.5.3)."
+      example: "Lighthouse / v0.1.5 (Linux x86_64)"
+    genesis_time:
+      type: integer
+      format: uint64
+      description: "The genesis_time configured for the beacon node, which is the time the Eth1.0 validator deposit smart contract has enough ETH staked (i.e. Eth2.0 begins)."
```

Removed the reference to the deposit contract because other potential eth2 chains might start without the connection to an eth1 chain.
```suggestion
      description: "The genesis_time configured for the beacon node, which is the unix time at which the Eth2.0 chain began."
```